components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dto/OAuth2IntrospectionResponseDTO.java

```diff
@@ -114,6 +116,11 @@
      */
     private String aut;
 
+    /*
+     * OPTIONAL. User object for the resource owner who authorized this token.
```

Comment:
Represents the resource owner who authorized this token.

Reply:
BTW what would be the behaviour for client credential grant type?

Reply:
Done with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1549/commits/0c7f5f1efe5bc0667cf986d6c4bc360f6c91f99c